src/plugins/i18n.js

```diff
@@ -1,10 +1,10 @@
-import Vue from 'vue';
-import VueI18n from 'vue-i18n';
+import Vue from "vue";
+import VueI18n from "vue-i18n";
 
 Vue.use(VueI18n);
 
-function loadLocaleMessages() {
-  const locales = require.context('./../locales', true, /[A-Za-z0-9-_,\s]+\.json$/i);
+const loadLocaleMessages = function () {
```

Comment:
what if we do 
`const loadLocaleMessages = () => {`

Reply:
It's changed!